### ISSUE TITLE
Integrate existing performance tests with CI/PRs

### ISSUE THREAD

**Issue by ktrott** [Netflix/falcor#258]
Get CI for perf tests

**Comment by ktrott** [comment]
@sdesai This is complete? Or we should break out into separate issues to detail the remaining work.

**Comment by sdesai** [comment]
No. The building blocks are all in place (devs can run perf tests, across all envs.), but we need to tie them together, and integrate them with CI builds.

@michaelbpaulson and I outlined the individual steps needed to close it out in December. I believe he has them captured somewhere, otherwise can re-define them.
